DirectEmployers/MyJobs
# remove overview page

## What's this PR do?
Removes 'Overview' page from manage users.

## Screenshots (if appropriate)
![screen shot 2017-09-29 at 2 23 20 pm](https://user-images.githubusercontent.com/25237191/31030020-cf2f89ae-a521-11e7-96a7-a7a9e2819a38.png)

## Where should the reviewer start?
Manage Users, the first view should be the User list. The tab should be active. 

## How should this be manually tested?
Click through tabs, make sure routes work. 
* I did not rename the routes, but would happy to do so

## PR Checklist

### Front-End


### Separation of Concerns


### Tests


### Migrations
